app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,49 @@
+@import "foundation_and_overrides";
+
+body {
+  background: url("Home_photo.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size<br>Avoid vendor prefixes.